docs/design/datacontracts/RuntimeTypeSystem.md

```diff
@@ -4,17 +4,64 @@ This contract is for exploring the properties of the runtime types of values on
 
 ## APIs of contract
 
-A `MethodTable` is the runtime representation of the type information about a value.  Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `MethodTableHandle` for querying the `MethodTable`.
+A `MethodTable` is the runtime representation of the type information about a value which is represented as a TypeHandle.
+Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `MethodTableHandle` for querying the `MethodTable`.
+Some other apis are available while using `TypeHandle` which is provided similarly to a MethodTable
+
 
 ``` csharp
 struct MethodTableHandle
 {
     // no public properties or constructors
 
-    internal TargetPointer Address { get; }
+    public TargetPointer Address { get; }
+}
+
+struct TypeHandle
```

Comment:
It's an opaque structure that can be used with some of the new apis. @lambdageek and I have been talking about simply using TypeHandle for all types instead of having a pair of opaque structures, and after finishing this up, I think that's the right approach. I was waiting for him to drop in on this before I made such a change.